tools/parse_soundfont.py

```diff
@@ -1033,28 +1038,28 @@ def process_oot_instruments(self):
                         oot_inst.enum = f"{presetheader.achPresetName.replace(' ', '_').replace('(', '_').replace(')', '_').upper()}"
                         oot_inst.envelope = curInst.envelope_enum
                         release = find_closest_index(curInst.release, release_values)
-                        oot_inst.releaserate = ctypes.c_int8(release).value
+                        oot_inst.releaserate = ctypes.c_uint8(release).value # release rates are UNSIGNED integers, not SIGNED integers owl...
```

Comment:
the audio compiling scripts expect a signed integer here, the script accounts for this so the binary value is the same as if it was unsigned when it compiles. don't ask me why its like that as i did not write the audio compiling scripts.